core/src/main/scala/akka/persistence/cassandra/query/EventsByTagStage.scala

```diff
@@ -234,12 +234,15 @@ import com.datastax.driver.core.utils.UUIDs
         stageState = StageState(QueryIdle, fromOffset, calculateToOffset(), initialTagPidSequenceNrs, None, bucketSize)
         if (log.isInfoEnabled) {
           log.info(
-            s"[{}]: EventsByTag query [${session.tag}] starting with EC delay {}ms: fromOffset {} toOffset {} initial state " + stageState,
+            s"[{}]: EventsByTag query [${session.tag}] starting with EC delay {}ms: fromOffset [{}] toOffset [{}]",
             stageUuid,
             settings.eventsByTagEventualConsistency.toMillis,
             formatOffset(fromOffset),
             toOffset.map(formatOffset))
         }
+        if (log.isDebugEnabled) {
```

Comment:
not needed, unless `stageState.tagPidSequenceNrs` does something heavy

Reply:
Yep, too much copy pasta